static/src/js/components/TemplateList/TemplateList.jsx

```diff
@@ -38,21 +40,21 @@ import Table from '@/js/components/Table/Table'
  */
 const TemplateList = () => {
   const { templateType } = useParams()
-  const { user } = useAppContext()
-  const { token } = user
+  const { providerId } = useAppContext()
+
+  const [cookies] = useCookies([MMT_COOKIE])
+  const { [MMT_COOKIE]: mmtJwt } = cookies
```

Comment:
Would it be more simple to have a `useMMTCookie` hook that just returned it, rather than needing to destructure everywhere?

Reply:
I thought about this, but its probably more code to create the hook as often as we use it.

Reply:
It was used 6 places, the hook made sense, so I implemented it already lol